pennylane/templates/embeddings.py

```diff
@@ -39,107 +110,42 @@ def AngleEmbedding(features, wires, rotation='X'):
             RY(f, wires=w)
-
     elif rotation == 'Z':
         for f, w in zip(features, wires):
             RZ(f, wires=w)
 
-    else:
-        raise ValueError("Rotation has to be `X`, `Y` or `Z`; got {}.".format(rotation))
-
-
-def AmplitudeEmbedding(features, wires, pad=False, normalize=False):
-    r"""Encodes :math:`2^n` features into the amplitude vector of :math:`n` qubits.
-
-    If the total number of features to embed are less than the :math:`2^n` available amplitudes,
-    non-informative constants (zeros) can be padded to ``features``. To enable this, the argument
-    ``pad`` should be set to ``True``.
-
-    The L2-norm of ``features`` must be one. By default, AmplitudeEmbedding expects a normalized
-    feature vector. The argument ``normalize`` can be set to ``True`` to automatically normalize it.
-
-    .. note::
-
-        AmplitudeEmbedding uses PennyLane's :class:`~pennylane.ops.QubitStateVector` and only works in conjunction with
-        devices that implement this function.
-
-    Args:
-        features (array): input array of shape ``(2**n,)``
-        wires (Sequence[int]): sequence of qubit indices that the template acts on
-        pad (Boolean): controls the activation of the padding option
-        normalize (Boolean): controls the activation of automatic normalization
-
-    Raises:
-        ValueError: if ``features`` or ``wires`` is invalid
-    """
-
-    if isinstance(wires, int):
-        wires = [wires]
-
-    features = np.array(features)
-
-    n_features = len(features)
-    n_amplitudes = 2**len(wires)
-
-    if n_amplitudes < n_features:
-        raise ValueError("AmplitudeEmbedding requires the size of feature vector to be "
-                         "smaller than or equal to 2**len(wires), which is {}; "
-                         "got {}.".format(n_amplitudes, n_features))
-
-    if pad and n_amplitudes >= n_features:
-        features = np.pad(features, (0, n_amplitudes-n_features), 'constant')
-
-    if not pad and n_amplitudes != n_features:
-        raise ValueError("AmplitudeEmbedding must get a feature vector of size 2**len(wires), "
-                         "which is {}; got {}. Use ``pad=True`` to automatically pad the "
-                         "features with zeros.".format(n_amplitudes, n_features))
-
-    # Get normalization
-    norm = 0
-    for f in features:
-        if isinstance(f, Variable):
-            norm += np.conj(f.val)*f.val
-        else:
-            norm += np.conj(f)*f
-
-    if not np.isclose(norm, 1):
-        if normalize:
-            features = features/np.sqrt(norm)
-        else:
-            raise ValueError("AmplitudeEmbedding requires a normalized feature vector. "
-                             "Set ``normalize=True`` to automatically normalize it.")
-
-    QubitStateVector(features, wires=wires)
-
 
 def BasisEmbedding(features, wires):
     r"""Encodes :math:`n` binary features into a basis state of :math:`n` qubits.
```

Comment:
On line 153, ``features=[0, 1, 0]`` might need to be changed such that it refers to an array e.g. ``features=array([0, 1, 0])`` (passing a list results in an error)